### DISCUSSION TITLE
If I have install `@babel/runtime-corejs3` do I still need to install `core-js@3` in my `package.json`?

### DISCUSSION THREAD

**Discussion by nyngwang** [babel/babel#13698] [Q&A] [answered]
I've described my problem on StackOverflow, here: https://stackoverflow.com/q/68878931/5290519.

Basically I want to know that If I ihave nstall `@babel/runtime-corejs3` do I still need to install `core-js@3` in my `package.json`?

**Comment by JLHwung** [comment, marked as answer]
You don't need to install `core-js@3` if you are using `@babel/transform-runtime`, unless you import it somewhere in your app sources. If you are setting up a new project, I would recommend you to try out https://github.com/babel/babel-polyfills#babel-polyfills, it works well with `preset-env`, see https://github.com/babel/babel-polyfills/blob/main/docs/migration.md for migration.
